main.js

```diff
@@ -0,0 +1,197 @@
+//global variables
+
+let board = document.querySelectorAll('div[id^="cell-"]');
```

Comment:
A clever way to select all the cells. I know this returns an array, but comments would help clarify your code and improve readability.